Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -675,7 +675,9 @@ In case of a multi-device environment, where Cilium's device auto-detection sele
 more than a single device to expose NodePort or a user specifies multiple devices
 with ``devices``, the XDP acceleration is enabled on all devices. This means that
 each underlying device's driver must have native XDP support on all Cilium managed
-nodes. In addition, for performance reasons we recommend kernel >= 5.5 for
+nodes. If you have an environment where some devices support XDP but others do not
+You can have XDP enabled on the supported devices by setting loadBalancer.mode to 
```

Comment:
Shouldn't this be `loadBalancer.acceleration` rather than `loadBalancer.mode`??

Reply:
Yes, good catch.